server/main.lua

```diff
@@ -111,79 +94,73 @@ end
 QBCore.Commands.Add("spikestrip", Lang:t("commands.place_spike"), {}, false, function(source)
     local src = source
     local Player = QBCore.Functions.GetPlayer(src)
-    if Player.PlayerData.job.name == "police" and Player.PlayerData.job.onduty then
-        TriggerClientEvent('police:client:SpawnSpikeStrip', src)
-    end
+    if not (Player.PlayerData.job.name == "police" and Player.PlayerData.job.onduty) then return end
+    TriggerClientEvent('police:client:SpawnSpikeStrip', src)
 end)
 
 QBCore.Commands.Add("grantlicense", Lang:t("commands.license_grant"), {{name = "id", help = Lang:t('info.player_id')}, {name = "license", help = Lang:t('info.license_type')}}, true, function(source, args)
     local src = source
     local Player = QBCore.Functions.GetPlayer(src)
-    if Player.PlayerData.job.name == "police" and Player.PlayerData.job.grade.level >= Config.LicenseRank then
-        if args[2] == "driver" or args[2] == "weapon" then
-            local SearchedPlayer = QBCore.Functions.GetPlayer(tonumber(args[1]))
-            if not SearchedPlayer then return end
-            local licenseTable = SearchedPlayer.PlayerData.metadata["licences"]
-            if licenseTable[args[2]] then
-                TriggerClientEvent('QBCore:Notify', src, Lang:t("error.license_already"), "error")
-                return
-            end
-            licenseTable[args[2]] = true
-            SearchedPlayer.Functions.SetMetaData("licences", licenseTable)
-            TriggerClientEvent('QBCore:Notify', SearchedPlayer.PlayerData.source, Lang:t("success.granted_license"), "success")
-            TriggerClientEvent('QBCore:Notify', src, Lang:t("success.grant_license"), "success")
-        else
-            TriggerClientEvent('QBCore:Notify', src, Lang:t("error.error_license_type"), "error")
-        end
-    else
-        TriggerClientEvent('QBCore:Notify', src, Lang:t("error.rank_license"), "error")
+    if not (Player.PlayerData.job.name == "police" and Player.PlayerData.job.grade.level >= Config.LicenseRank) then
```

Comment:
```suggestion
    if Player.PlayerData.job.name ~= "police"  or Player.PlayerData.job.grade.level < Config.LicenseRank then
```

A bit easier to read.

Reply:
Completely forgot about the ```~=``` operand, will start using afterwards, will commit changes featuring this